hazelcast.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Hazelcast - a streaming and memory-first application platform
+After=syslog.target network.target
+
+[Service]
+SuccessExitStatus=143
```

Comment:
For apt versoin I've set:
```
User=hazelcast
Group=hazelcast
```
and run:
```shell
groupadd -r hazelcast
useradd -r -g hazelcast -d /usr/lib/hazelcast -s /sbin/nologin hazelcast
chown -R hazelcast:hazelcast /usr/lib/hazelcast
```
in `postinst`. What do you think about running the service as dedicated service user?

Reply:
I thought about that and there were some open questions that it seemed we need to think through:

- does this handle that the user/group already exists?
- do we remove the user/group after uninstall?
- what should the permissions be? what happens when you run `hz start` from users account (but this is actually a question also when it's owned by the root user)
- in the RPM guide I saw some stuff should be done so when you install the package on multiple machines the user/group will end up with the same id, but maybe we don't need that